<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="TitleMainWindow">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestor de contraseÃ±as</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GenerarPasswordButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Generar contraseÃ±a</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ShowPassword">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="SiteAccount">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="EmailAccount">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="PasswordAccount">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddAccount">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar cuenta</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
